libraries/cbor/src/values.rs

```diff
@@ -67,37 +70,22 @@ impl Value {
 
     pub fn type_label(&self) -> u8 {
         match self {
-            Value::KeyValue(key) => key.type_label(),
+            Value::Unsigned(_) => 0,
```

Comment:
Wouldn't it make sense here to use an enum or something alike for the labels?
This would also make clearer the calls to `self.start_item(...)` in the CBOR writer.

Reply:
Good catch with the writer. It should use the existing `type_label` function. Fixed. This function should be inlined later into the enum declaration using the still unstable enum discriminant.

Reply:
Ok then we're good as it is I think.
Replacing the numeric values for the type by an enum will improve readability in the CBOR reader and avoid potential mistakes if new types need to be added. This can be deferred until the feature gets more stable.